jbpm-installer/src/main/resources/build.properties

```diff
@@ -25,13 +25,13 @@ jBPM.url=https://repository.jboss.org/nexus/service/local/artifact/maven/redirec
 # for example: 
 #<RELEASE>
 # for EAP7:
-#   jBPM.console.url=http://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/kie-wb-distribution-wars/${jBPM.version}/kie-wb-distribution-wars-${jBPM.version}-eap7.war
+#   jBPM.console.url=http://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/kie-wb/${jBPM.version}/kie-wb-${jBPM.version}-eap7.war
 # for WildFly10:
-#   jBPM.console.url=https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/kie-wb-distribution-wars/${jBPM.version}/kie-wb-distribution-wars-${jBPM.version}-wildfly10.war
+#   jBPM.console.url=https://repository.jboss.org/nexus/content/groups/public-jboss/org/kie/kie-wb/${jBPM.version}/kie-wb-${jBPM.version}-wildfly10.war
 # or:
 #<SNAPSHOT>
-#   jBPM.console.url=https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=kie-wb-distribution-wars&v=${jBPM.version}&e=war&c=wildfly10
-jBPM.console.url=https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=kie-wb-distribution-wars&v=${jBPM.version}&e=war&c=wildfly10
+#   jBPM.console.url=https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=kie-wb&v=${jBPM.version}&e=war&c=wildfly10
+jBPM.console.url=https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.kie&a=kie-wb&v=${jBPM.version}&e=war&c=wildfly10
```

Comment:
This could probably use a fix to something like `jBPM.workbench.url`, but that's out of the scope of this PR.